cmd/utils_test.go

```diff
@@ -354,3 +354,37 @@ func Test_sliceContains_Fail(t *testing.T) {
 		t.Errorf("Slice a should not have b")
 	}
 }
+
+func Test_generateGraph(t *testing.T) {
+
+	/*
+
```

Comment:
nit: please remove this extra line to maintain consistency with other comments which show a graph 🙂